### ISSUE TITLE
Abnormal cellular composition of broncho-alveolar fluid 

### ISSUE THREAD

**Issue by mgriese** [obophenotype/human-phenotype-ontology#5555]
**Preferred term label:**
Abnormal cellular composition of broncho-alveolar fluid 

**Synonyms**
Abnormal cell differential of broncho-alveolar fluid 

**Definition (free text, please give PubMed ID)**
Deviation from the commonly in healthy people observe cellular distribution. Normal are macrophages >80%, Lymphocytes <15%, Neutrophils <3%, Eosinophils <0.5%, Mast cells <0.5%
Costabel ISBN0412792702; PMID: 3396403; 10678650; 

**Parent term (use hpo.jax.org/app)**
Abnormal broncho-alveolar lavage fluid morphology 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
@GrieseAG
@Katrinknoflach

**Comment by GrieseAG** [comment]
`pulmo`
